airflow/helm/airflow/values.yaml.tpl

```diff
@@ -160,11 +164,13 @@ airflow:
 
   dags:
     gitSync:
-      enabled: true
+      enabled: {{ .Values.gitSyncEnabled | ternary "true" "false" }}
```

Comment:
Ok I've tested this locally and just realized this setting will break existing airflow installs.  We'd probably need to set it to a default-falsy setting, like:

```
{{ if not .Values.gitSyncDisabled }}
  dags:
    gitSync:
       ....
{{ end }}
```

Since people won't have the gitSyncEnabled flag set in their context by default.

Reply:
Good point, agreed